tartiflette/types/exceptions/tartiflette.py

```diff
@@ -35,13 +37,16 @@ def coerce_value(
             pass
         except TypeError:
             pass
-        return {
+        errors = {
             "message": self.user_message
             if self.user_message
             else self.message,
             "path": path or self.path,
             "locations": computed_locations,
         }
+        if self.extensions:
+            errors["extensions"] = self.extensions
```

Comment:
I would wrap `self.extensions` to ensure it returns a dict by doing `errors["extensions"] = dict(self.extensions)` or `errors["extensions"] = {**self.extensions}`. Your thoughts @abusi and @Maximilien-R ? :)

Reply:
As the content of extension parameter is user responsability and ownership, I prefer not to touch it and keeping it as is.
If the output of a request doesn't fit the need of the extensions field for user, it's the user responsability to change it in it's code by giving ttftt the value he wants in the results.

Reply:
@abusi while I agree with you I think users also expect ttftt to respect the norm and perform some kind of validation when possible. So I think @achedeuzot 's proposal is good and I think we should do it.

Reply:
IMHO to defend against bad users we're going to hinder the perf of the good ones BUT because of the norm argument; i'll adhere to your POV. You can make sure it's a dict here.

Reply:
done